src/index.html

```diff
@@ -6,13 +6,89 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__img"
+        src="./images/imac.jpeg"
+        alt="imac"
+      />
+      <h1 class="product-card__title">
+        APPLE A1419 iMac 27" Retina
+        <br/>
+        5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="product-card__text"
+      >Product code: 195434
+      </p>
+
+      <div
+      class="reviews"
+      >
+        <div
+        class="reviews__stars"
+        >
+          <div
+            class="reviews__star-item"
+            >
```

Comment:
Fix it everywhere and add the modifier classes
```suggestion
        <div class="stars stars--4">
          <div class="stars__star">
```

Reply:
I am fixed